guides/07_FAQs.md

```diff
@@ -145,6 +145,24 @@ The possible reasons are as follows:
 2. The close code `4410` from the Chime backend indicates that an attendee has attempted to join an already-ended meeting. The Chime SDK for JavaScript throws an error with the status code `MeetingEnded`.
 3. The close code between `4500` and `4599` (inclusive) indicates an internal server error in the Amazon Chime backend. In this case, the Chime SDK for JavaScript throws an error with the status code `SignalingInternalServerError`. Please create a GitHub issue including the Chime SDK browser logs.
 
+### When does the Amazon Chime SDK retry the connection? Can I customize this retry behavior?
+The Amazon Chime SDK for JavaScript retries the connection in the following situations.
+
+* The SDK misses consecutive pong messages from the Chime server.
+* The SDK detects a high rate of audio packet loss.
+* The SDK experiences a significant audio delay.
+* The SDK encounters a retryable error during the session.
+
+The SDK uses [ConnectionHealthPolicyConfiguration](https://github.com/aws/amazon-chime-sdk-js/blob/master/src/connectionhealthpolicy/ConnectionHealthPolicyConfiguration.ts) to trigger a reconnection. We recommend using the default configuration, but you can also provide the custom ConnectionHealthPolicyConfiguration object to change this behavior. 
+
+* Configure [missed pong threshold](https://github.com/aws/amazon-chime-sdk-js/blob/master/src/connectionhealthpolicy/ConnectionHealthPolicyConfiguration.ts#L24): The current missed pong threshold is set to 4 in JS SDK. For example, you can set the missed pongs upper threshold to zero to force restart the session.
```

Comment:
Instead of these points, just recommend adding the code example from the previous comment. Customers will have challenges updating these values because we don't have any example of overriding the health policy configuration.

Reply:
Yeah, make sense. I will add the code example. Can I also keep these description text along with the code examples?